### PR TITLE
Chore: Remove brui reset style. Add proper style to AF.scss

### PR DESCRIPTION
**Pull request by JustinHoldstock**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @JustinHoldstock has signed the CLA. Thanks for the pull request!

File: src/components/base.scss [hunk 1]
```diff
@@ -5,8 +5,6 @@
 .be {
     @include reset;
 
-    @import '~box-react-ui/lib/styles/reset';
```
Comment (priyajeet) [inline src/components/base.scss]: Hmm, wasn't this already resetting scoped to .be?

**Comment by boxcla** [comment]
# Webpack Bundle Size Github Bot

| Bundle        | Diff | Percentage Diff | Before | After |
| ------------- |:----:|:---------------:|:------:|:-----:|
| Total | 0 B | 0.00% | 0 B | 0 B |

Compared against previously merged PR #278

**Powered by [webpack-bundle-size-github-bot](https://github.com/wenboyu2/webpack-bundle-size-github-bot/)**

File: src/components/base.scss (hunk 1 above)
Comment (JustinHoldstock) [reply src/components/base.scss]: Via this import, yes, but it wasn't playing well with Details tab. Appears to be the only thing activity feed was using was the `ol, ul` style added to ActivityFeed.scss

**Pull request merged**